stable_baselines3/common/logger.py

```diff
@@ -66,6 +67,22 @@ def __init__(self, image: Union[th.Tensor, np.ndarray, str], dataformats: str):
         self.dataformats = dataformats
 
 
+class HParam:
+    """
+    Hyperparameter data class storing hyperparameters and metrics in dictionnaries
+
+    :param hparam_dict: key-value pairs of hyperparameters to log
+    :param metric_dict: key-value pairs of metrics to log
+        A non-empty metrics dict is required to display hyperparameters in the corresponding Tensorboard section.
+    """
+
+    def __init__(self, hparam_dict: Dict[str, Union[bool, str, float, int, None]], metric_dict: Dict[str, Union[float, int]]):
+        self.hparam_dict = hparam_dict
+        if not metric_dict:
+            warnings.warn("`metric_dict` must not be empty to display hyperparameters to the HPARAMS tensorboard tab.")
```

Comment:
why not an error in that case?

Reply:
True, nothing will break but the hparams tab will be empty. I have added an exception.